Cargo.toml

```diff
@@ -19,9 +19,10 @@ codecov = { repository = "jmagnuson/linemux", branch = "master", service = "gith
 futures-util = { version = "0.3", default-features = false, features = ["std"] }
 notify = "5.0.0-pre.4"
 pin-project-lite = "0.1"
-tokio = { version = "0.2", features = ["fs", "io-util", "stream", "sync", "time"] }
+tokio = { version = "1.0", features = ["fs", "io-util", "sync", "time"] }
+tokio-stream = "0.1"
```

Comment:
Yeah, it's too bad they had to break this out into a separate crate, but understand the not wanting to break backwards compat, etc.

Not sure why I never thought to do this, but what do you think about just using [`futures_util::stream::Stream`](https://docs.rs/futures-util/0.3.12/futures_util/stream/trait.Stream.html)? I guess the only concern would be if `tokio-stream` went and created their own `Stream` variant, but seems unlikely at this stage? Fine as-is, just throwing out a suggestion.

Reply:
This does seem to work just fine. That means we don't have to pull in a new dependency, yay!